compiler/rustc_error_codes/src/error_codes/E0069.md

```diff
@@ -12,3 +12,4 @@ fn foo() -> u8 {
 
 Since `return;` is just like `return ();`, there is a mismatch between the
 function's return type and the value being returned.
+So Use `return ();` Insted.
```

Comment:
Using `return ();` instead would not fix the error in the example, since `return;` and `return ();` mean the same thing. The program would need to return a `u8` value, like `return 0u8;`.